test/test_masks_to_bounding_boxes.py

```diff
@@ -0,0 +1,43 @@
+import os.path
+
+import PIL.Image
+import numpy
+import pytest
+import torch
+
+import torchvision.ops
+
+ASSETS_DIRECTORY = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets")
+
+
+@pytest.fixture
+def labeled_image() -> torch.Tensor:
+    with PIL.Image.open(os.path.join(ASSETS_DIRECTORY, "labeled_image.png")) as image:
+        return torch.tensor(numpy.array(image, numpy.int))
+
+
+@pytest.fixture
+def masks() -> torch.Tensor:
+    with PIL.Image.open(os.path.join(ASSETS_DIRECTORY, "masks.tiff")) as image:
```

Comment:
Do you think it would be possible to write a test without the need for new images and hard-coded coordinates?

Ideally, we could generate random masks and have a super simple version of `masks_to_boxes` which we could use as the reference implementation?

Reply:
@NicolasHug a friendly bump